server/channels/app/export.go

```diff
@@ -447,7 +482,9 @@ func (a *App) exportAllUsers(ctx request.CTX, job *model.Job, writer io.Writer,
 	cnt := 0
 	profilePictures := []string{}
 	for {
-		users, err := a.Srv().Store().User().GetAllAfter(1000, afterId)
+		var users []*model.User
+		var err error
+		users, err = a.Srv().Store().User().GetAllAfter(1000, afterId)
```

Comment:
What was the reason for this change?

Reply:
Good catch, it must have been some previous refactoring. I have reverted it.